.coveragerc

```diff
@@ -6,6 +6,7 @@ omit =
     ngmix/admom/*_nb.py
     ngmix/gmix/*_nb.py
     ngmix/pixels/*_nb.py
+    ngmix/tests/*
```

Comment:
It is a good idea to include tests in coverage since they should be at 100% to ensure we actually execute all of them.